modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
+Start xref:create-api-specification-studio.adoc[an API specification project in Studio and sync it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
 
-You can also import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and sync your changes with Design Center:
+Import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and sync your changes with Design Center:
 
 . Design an API specification in Design Center. +
-You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Code Editor].
+You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Text Editor].
 . Import the API specification from Design Center into Studio. +
 You can import the API specification as an editable asset by xref:sync-api-projects-design-center.adoc[Synchronizing an API Specifications with Design Center].
 
 You can also edit an API specification that you imported as an API dependency in your project:
 
 . Import the API specification xref:import-api-specification-exchange.adoc[from Exchange to Studio]. +
-Your API specification is imported to Anypoint Studio as a dependency. This means that your Mule application project now holds a reference to the specific API version that you're implementing. Studio monitors your API and notifies you when there is a new version available.
+Studio imports your API specification as a dependency. This means that your Mule application project now holds a reference to the specific API version that you're implementing. Studio monitors your API and notifies you when there’s a new version available.
 . Select the API specification imported in your project and xref:sync-imported-api-specifications-design-center.adoc[Edit your API specification from Studio] to later sync your changes to Design Center.
 
-Studio uses the EGit plugin only for the VCS feature of editing API specifications offline. Studio does not support the EGit plugin for Mule application projects that you might track using your own VCS.
+[NOTE]
+Studio uses the EGit plugin for the VCS feature of editing API specifications offline. Studio doesn't support the EGit plugin for Mule application projects that you track using your own VCS.
```

Comment:
Is there a way to streamline "VCS feature of editing..."? Do you need to mention VCS in that sentence?

Reply:
Let's see if my rewording works.
Thanks!